### ISSUE TITLE
Youtube preview play crashes tab

### ISSUE THREAD

**Issue by Legion495** [MrAlex94/Waterfox#2627]
### What happened?

Hi! Wanted to hopefully switch to Waterfox on my machines.

I run Fedora 35 with waterfox and firefox (on my Laptop).
Waterfox G4 has been installed from hawkeye116477 repository.

If I hover over YT tiles it causes the tab to crash. 
I tried in private mode and without extensions of course.

The issue does not show up on firefox on the same machine. 
Wayland and standard waterfox behave the same way on this specific machine.
I noticed it does not happen on every video. For me many Markiplier Videos seem affected.
I also have a Manjaro System with Waterfox installed but it works totally normal there. So I assume this might be distro related. 
The biggest issue I have. I would like to know how to enable debug and create logs to maybe see the cause. I would appreciate input on that. 

I forgot to add yes Manjaro runs KDE and Fedora Gnome. 

### Reproducible?

- [ ] I have tested this on Firefox.

### Version

G4

### What platform are you seeing the problem on?

Linux

**Comment by adamp01** [comment]
@Legion495  - Thanks for raising this. 

Opening up the browser console with `Ctrl+Shift+J` should allow you to see some logs that we can use to identify what is causing this.

Thanks

**Comment by Legion495** [comment]
![image](https://user-images.githubusercontent.com/15214766/161389286-91ad9418-5a5a-4b59-99f9-b3a29316dc6a.png)
![image](https://user-images.githubusercontent.com/15214766/161389502-0f09b9d3-dfbc-4241-bcc6-fd36d8034c56.png)

All of that looks rather generic. Everything edited again because I recorded a video. I made sure to wipe the browser clean.
And I highly doubt it can cause issues but I ran FF in the background during these tests. It did not help before not running FF but let me know if there is a indication in that direction.

Now I do not know if locals can cause issues. I use a german keyboard layout "Austria no dead keys", so in locals that is like "AT-EN" or something (because the browser warns about language) but the OS is kept in English. I mention it because I think there is a minority doing that.
Here is the video:
https://www.youtube.com/watch?v=yMvzMyxlgAU

Now there was something new. Upon reload it threw many errors at me.
Hopefully the video processes up to FHD. I will leave the browser in this state.
![image](https://user-images.githubusercontent.com/15214766/161390152-0e0b7322-b429-40e2-bccc-9bdb7f489268.png)
![image](https://user-images.githubusercontent.com/15214766/161390567-c5a13201-95bc-4ebb-8513-12b3913a9a2d.png)

**Comment by adamp01** [comment]
Thanks for sending over logs and for the video. Unfortunately there is nothing related to the tab crashing appearing so it's going to be really hard to figure this one out. It would be good to know what version of Firefox you tested where it was working, and would also be good to know if you test Firefox ESR91 does it work or fail?

If you're using a version of FF > 91 and it works but version 91 fails, there may be some commits we can pull through to fix this.

**Comment by Legion495** [comment]
So normally I run the rolling release:
![image](https://user-images.githubusercontent.com/15214766/162639310-99812d3a-3f67-4142-a964-608748a00340.png)
But I went ahead and downloaded the ESR91:
![image](https://user-images.githubusercontent.com/15214766/162639575-8175da15-5641-49f4-b42d-1b92926fa133.png)

I have to report no issues on this version either... maybe I will do a reinstall, just in case.
Update:Did that and immediately could crash the tab again.
If you got more ideas let me know.

Edit once again: The repository install is at fault!
I downloaded it straight from waterfox webpage and have zero issues. But sadly it does not integrate with the OS that great then.
Why and how is beyond me...

We may close this then.
I hope a flatpak will be available eventually.
